lib/src/widgets/chat/ds_image_message_bubble.dart

```diff
@@ -0,0 +1,175 @@
+                                    child: Row(
+                                      children: [
+                                        IconButton(
+                                          padding: EdgeInsets.zero,
+                                          onPressed: () =>
+                                              Navigator.of(context).pop(),
+                                          icon: const Icon(
+                                            Icons.arrow_back_ios,
+                                            color: DSColors.neutralLightSnow,
+                                          ),
+                                        ),
+                                        Expanded(
+                                          child: ListTile(
+                                            contentPadding: EdgeInsets.zero,
+                                            leading: DSUserAvatar(
+                                              text: appBarText,
+                                            ),
+                                            title: DSHeadlineSmallText(
+                                              text: appBarText,
+                                              color: DSColors.neutralLightSnow,
+                                            ),
+                                          ),
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                ),
+                              ),
+                              body: GestureDetector(
+                                onTap: () => _controller.showAppBar(),
+                                child: Container(
+                                  width: MediaQuery.of(context).size.width,
+                                  height: MediaQuery.of(context).size.height,
+                                  padding: const EdgeInsets.all(8.0),
+                                  color: Colors.black,
+                                  child: PinchZoom(
+                                    resetDuration:
+                                        const Duration(milliseconds: 100),
+                                    child: DSCachedNetworkImageView(
+                                      url: url,
+                                      fit: BoxFit.contain,
+                                    ),
+                                  ),
+                                ),
+                              ),
+                            );
+                          },
+                        ),
+                      ),
+                    );
+                  },
+                );
+              }
+            },
+            child: DSCachedNetworkImageView(
+              width: 240.0,
+              height: 240.0,
+              url: url,
+              placeholder: (context, url) => const Padding(
```

Comment:
As described in Effective Dart: Style, if the anonymous function has unused parameters, it's preferable to use underscores instead of an actual name.

```dart
placeholder: (_, __) => const Padding(...)
```

https://dart.dev/guides/language/effective-dart/style#prefer-using-_-__-etc-for-unused-callback-parameters